libbeat/autodiscover/builder/plugin.go

```diff
@@ -23,7 +23,7 @@ func init() {
 	p.MustRegisterLoader(pluginKey, func(ifc interface{}) error {
 		b, ok := ifc.(builderPlugin)
 		if !ok {
-			return errors.New("plugin does not match processor plugin type")
+			return errors.New("plugin does not builder processor plugin type")
```

Comment:
does not have builder ...?

Reply:
reflects sleep deprivation :) fixing. apologies for poor grammar